gateway/Makefile

```diff
@@ -1,20 +1,22 @@
 include ../config.mk
 
-.PHONY: build push deploy
+.PHONY: envoy-xds-config envoy-config deploy-no-xds deploy
```

Comment:
Looks like there's no `envoy-config` anymore. Is it missing because it looks like it's expected in the deployment. Ah I see, `envoy.yaml` is static in the repo.

Reply:
Cleaned this up.